UGI-216004-main/PhotoEnhancer/PhotoEnhancer/Filters/GammaСolorСorrectionParameters.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PhotoEnhancer
+{
+    public class GammaСolorСorrectionParameters : IParameters
+    {
+        public double CoefficientR { get; set; }
+        public double CoefficientG { get; set; }
+        public double CoefficientB { get; set; }
+        public ParameterInfo[] GetDecription()
+        {
+            return new[]
+            {
+                new ParameterInfo()
+                {
+                    Name = "Коэффициент для канала R",
+                    MinValue = 0.2,
+                    MaxValue = 5,
+                    DefaultValue = 1,
+                    Increment = 0.01
+                },
+                new ParameterInfo()
+                {
+                    Name = "Коэффициент для канала G",
+                    MinValue = 0.2,
+                    MaxValue = 5,
+                    DefaultValue = 1,
+                    Increment = 0.05
+                },
+                new ParameterInfo()
+                {
+                    Name = "Коэффициент для канала B",
```

Comment:
Название параметров не совсем удачные. Ведь вы не умножаете яркость канала на это число, а производите иное преобразование. Термин[ "гамма-коррекция"](https://ru.wikipedia.org/wiki/%D0%93%D0%B0%D0%BC%D0%BC%D0%B0-%D0%BA%D0%BE%D1%80%D1%80%D0%B5%D0%BA%D1%86%D0%B8%D1%8F) является общеупотребимым.
Лучше так
```suggestion
                    Name = "Гамма канала R",
                    MinValue = 0.2,
                    MaxValue = 5,
                    DefaultValue = 1,
                    Increment = 0.01
                },
                new ParameterInfo()
                {
                    Name = "Гамма канала G",
                    MinValue = 0.2,
                    MaxValue = 5,
                    DefaultValue = 1,
                    Increment = 0.05
                },
                new ParameterInfo()
                {
                    Name = "Гамма канала B",
```